qr-link/src/user_data.rs

```diff
@@ -24,6 +24,8 @@ pub struct UserData {
     pub user_centric_signup: bool,
     /// A unique UUID that the Orb will use to send messages to the app through Orb Relay.
     pub orb_relay_app_id: Option<String>,
+    /// Whether the Orb should perform the age verification. If the token exists we skip the age verification.
+    pub bypass_age_verification_token: Option<String>,
```

Comment:
why is this a `String`?
if it should be `bool`, what about a rename to `bypass_age_verification`

Reply:
I think I got my answer from `...request from POP team`.